development/compiling/compiling_for_linuxbsd.rst

```diff
@@ -98,13 +98,22 @@ Start a terminal, go to the root dir of the engine source code and type:
 
 ::
 
-    scons -j$(nproc) platform=linuxbsd
+    scons -j8 platform=linuxbsd
 
 A good rule of thumb for the ``-j`` (*jobs*) flag, is to have at least as many
 threads compiling Godot as you have cores in your CPU, if not one or two more.
-```nproc``` can be used to automatically use the number of cores your system has.
 Feel free to add the ``-j`` option to any SCons command you see below.
 
+You can automatically use all available cores with command subtitution.
+
+On Linux you can use ``nproc``:
+::
+    scons -j$(nproc)
+    
+On BSD you can use ``sysctl -n hw.ncpu``:
+::
+    scons -j$(sysctl -n hw.ncpu)  
+
```

Comment:
```suggestion
You can automatically use all available CPU threads with command subtitution.

On Linux, you can use ``nproc``::

    scons -j$(nproc)
    
On *BSD, you can use ``sysctl -n hw.ncpu``::

    scons -j$(sysctl -n hw.ncpu)  

```

Also, doesn't *BSD have `hw.logicalcpu` to get the number of CPU threads (rather than cores)?

Reply:
Tried `sysctl -n hw.logicalcpu` on a OpenBSD 6.7 VM and GhostBSD (FreeBSD 12.2) VM and it didn't work.
I'm not a BSD user so `hw.ncpu` is what I found searching online.